tests/test_other.py

```diff
@@ -9452,3 +9452,14 @@ def test_SYSCALL_DEBUG(self):
   def test_LIBRARY_DEBUG(self):
     self.set_setting('LIBRARY_DEBUG')
     self.do_runf(path_from_root('tests', 'hello_world.c'), '[library call:_fd_write: 0x1')
+
+  def test_SUPPORT_LONGJMP_executable(self):
+    stderr = self.run_process([EMCC, path_from_root('tests', 'core', 'test_longjmp.c'), '-s', 'SUPPORT_LONGJMP=0'], stderr=PIPE, check=False).stderr
+    self.assertContained('error: longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=0, or remove uses of it in the project)',
+                         stderr)
+
+  def test_SUPPORT_LONGJMP_object(self):
+    self.run_process([EMCC, path_from_root('tests', 'core', 'test_longjmp.c'), '-s', 'SUPPORT_LONGJMP=0', '-c', '-o', 'a.o'])
+    stderr = self.run_process([EMCC, 'a.o', '-s', 'SUPPORT_LONGJMP=0'], stderr=PIPE, check=False).stderr
+    self.assertContained('error: longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=0, or remove uses of it in the project)',
+                         stderr)
```

Comment:
Do we need two different tests here?  I mean internally the object will be created either way.

Also SUPPORT_LONGJMP is ignored during compile phase so only needed at link time isn't it?

Reply:
The PR makes the flag be noticed during compile time, as it doesn't send the pass to the backend when longjmp is disabled (i added this after the first comment, so maybe you viewed before that).

But yeah, the test does seem redundant as it is. I'll change it to cover a corner case of compiling with support but linking without.